inst/include/Rcpp/platform/compiler.h

```diff
@@ -56,6 +43,13 @@
     #if GCC_VERSION >= 40600
         #define IS_GCC_460_OR_LATER
     #endif
+    // Inform users of dated compiler (e.g. <= RHEL6 )
```

Comment:
(I should add that RcppArmadillo now actually fails when this happens, but then again Conrad left me no choice. It has 246 rev.deps and mayhem may well ensue ...)